test/resume/TcpResumeClient.cpp

```diff
@@ -31,6 +32,39 @@ class Callback : public AsyncSocket::ConnectCallback {
 };
 }
 
+class ClientResumeStatus : public ClientResumeStatusCallback {
+public:
+  void onResumeOk() override {
+    std::cout << "resumed OK" << "\n";
+  }
+
+  void onConnectionError(folly::exception_wrapper ex) override {
+    std::cout << "resume connection error " << ex.what() << "\n";
+  }
+};
+
+class GuidGenerator {
+public:
+  // method to generate a UUID for the Resume Token
+  void static generateAndFill(ResumeIdentificationToken& token) {
```

Comment:
should we turn ResumeIdentificationToken into a class and integrate methods of GuidGenerator into it?